### PR TITLE
feat: Adding support for manifest strategy paths to be files

### PR DESCRIPTION
**Pull request by TigerC10**

### CODE REVIEW

**Comment by TigerC10** [comment]
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [X] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/release-please/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [X] Ensure the tests and linter pass
- [X] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #1366 🦕

**Comment by ScottGuymer** [comment]
I have found a similar use case where I would like to version and release single files.

In this case it is for GitHub Reusable Workflows. I have a number of shared workflows that I keep in a repo but GitHub mandate that they must all live in the `.github/workflows` folder and cant be nested or moved from there.

**Comment by ScottGuymer** [comment]
@chingor13 is this something that could move forward?

**Comment by ScottGuymer** [comment]
@yoshi-nodejs can this be reviwed?

**Comment by npalm** [comment]
@yoshi-nodejs would be great when the module can support file-based releases.

**Comment by TigerC10** [comment]
As my fork was fairly out of date, I've updated my fork to be in sync with the latest upstream.  No conflicts during update sync.

File: src/manifest.ts [hunk 1]
```diff
@@ -1208,7 +1211,8 @@ export class Manifest {
         const strategy = await buildStrategy({
           ...config,
           github: this.github,
-          path,
+          path:
+            existsSync(path) && lstatSync(path).isFile() ? dirname(path) : path,
```
Comment (chingor13) [inline src/manifest.ts:1215]: There is no cloned git repo so these `fs` commands are all going to return `false` all the time.

**Review by chingor13 — CHANGES_REQUESTED** [review]
This change of behavior will need tests to make sure it works (it currently will not work because the `fs` functions won't work on GitHub repo paths when there is no cloned version on the filesystem).

**Pull request closed without merging**